system/include/emscripten/wasmfs.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#pragma once
```

Comment:
Do these declarations need to be in a user-visible header or can we keep them in a private wasmfs header (or even just add the declarations to the relevant .cpp file)?

Reply:
I misread which functions these are!